pkg/daemon/ceph/osd/volume.go

```diff
@@ -205,7 +199,10 @@ func (a *OsdAgent) configureCVDevices(context *clusterd.Context, devices *Device
 
 	// List THE configured OSD with ceph-volume raw mode
 	if a.clusterInfo.CephVersion.IsAtLeast(cephVolumeRawModeMinCephVersion) && !lvBackedPV {
-		block = fmt.Sprintf("/mnt/%s", a.nodeName)
+		// When the block is encrypted we need to list against the encrypted device mapper
+		if !isEncrypted {
+			block = fmt.Sprintf("/mnt/%s", a.nodeName)
```

Comment:
Where is `block` set when it's encrypted?

Reply:
It's returned by the c-v list call.